### PR TITLE
Updated ipld document fix issues update previous field

### PR DESCRIPTION
**Pull request by jonnycrunch**
link for previous to intermediate file explained <cid>/previous/previous goes to genesis document.  Proof is on ethereum blockchain and not fully implemented.   signature updated to use DID "did:ipid:<cid>".  In future this needs more discussion.

### CODE REVIEW

**Comment by jonnycrunch** [comment]
@kimdhamilton @christianlundkvist @AnthonyRonning see updated document with fixed `previous`

File: draft-documents/ipld_did_documents.md
Comment (kimdhamilton) [inline draft-documents/ipld_did_documents.md:136]: The last sentence of the paragraph on line 136 is still a little confusing; from the phrasing, I expect to see that as `previous` in Example 2. However, your subsequent text clarifies how it fits together. I suggest a slight rewording of the last sentence (see bolded area)

This entire DID document when added to IPFS as IPLD has a CID of `zdpuAqiExr6k4AbWF6BuGkgUbVMZ7jbJyNvRz9z9yyRBxosPi` and will be used as the `previous` field in the subsequent updated DID document **`zdpuAvqt1YTeAdcyyncBDdJLVgKyZNbHm17rBEJNqFzVbg24B` (discussed below).**

**Review by kimdhamilton — APPROVED** [review]
Thanks for these changes @jonnycrunch -- very clear to me.

I think we'll still have to deal with the following 3 issues, but I don't think the paper needs to be blocked on them. After you merge, I can add them as "open issues"/TBD with details. 

These are things we likely need feedback from CCG on: some can fit into the existing DID spec and some may belong in an IPID-specific method spec.

Issues:
- `signature` -> `proof` renaming from DID data model (see [https://w3c-ccg.github.io/did-spec/#proof-optional](https://w3c-ccg.github.io/did-spec/#proof-optional))
- what happens with additional `proof` field; can this be a chained proof?
- how does the `message` field fit in?

**Pull request merged**
